lib/widgets/drive.coffee

```diff
@@ -34,6 +38,17 @@ getDrives = ->
 			async.reject drives, drivelist.isSystem, (results) ->
 				return callback(null, results)
 
+cleanupList = (previous,currrent) ->
+	removedDrive =
+		name: "#{currrent.device} (#{currrent.size}) - #{currrent.description}"
```

Comment:
Can you refactor this interpolation in a new function, maybe called `driveToString()` or something like that? This same logic is repeated three times on this files, which means that is very easy to get them out of sync, and mess all the equality checks.